src/Reader/readpar.f90

```diff
@@ -978,129 +797,20 @@ SUBROUTINE readdr(IO, EQN, DISC, BND, IC, MPI)
      logError(*) 'Dynamic rupture model file "', trim(ModelFileName), '" does not exist.'
      call exit(134)
     endif
-    !
-    DISC%DynRup%ModelFileName = ModelFileName
```

Comment:
should that be removed?

Reply:
Yes, we initialize everything in Fortran, so there is no need to store the model file name in Fortran.